src/main/java/com/ericsson/eiffel/remrem/semantics/clone/PrepareLocalEiffelSchemas.java

```diff
@@ -0,0 +1,74 @@
+package com.ericsson.eiffel.remrem.semantics.clone;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+import org.eclipse.jgit.api.Git;
+
+import com.ericsson.eiffel.remrem.semantics.schemas.EiffelConstants;
+import com.ericsson.eiffel.remrem.semantics.schemas.LocalRepo;
+import com.ericsson.eiffel.remrem.semantics.schemas.SchemaFile;
+
+/**
+ * This class is used to Clone the Eiffel Repo from github
+ * 
+ * @author xdurvak
+ */
+
+public class PrepareLocalEiffelSchemas {
+
+	/**
+	 * this method is used to clone Eiffel repository from github.
+	 * 
+	 * @param repoURL-
+	 *            repository url to clone as an input parameter
+	 * @param branch-
+	 *            specific branch name as an input parameter
+	 * @param localEiffelRepoPath
+	 *            - destination path to place the cloned repo as an input
+	 *            parameter
+	 */
+	private static void cloneEiffelRepo(String repoURL, String branch, File localEiffelRepoPath) {
+		try {
+			System.out.println("Latest Schema's Updating...");
+			if (!localEiffelRepoPath.exists()) {
+				Git.cloneRepository().setURI(repoURL).setBranch(branch).setDirectory(localEiffelRepoPath).call();
+			}
+		} catch (Exception e) {
+			e.printStackTrace();
+		}
+	}
+
+	public static void main(String[] args) throws IOException {
+
+		// IsClonable checking from build.gradle
+		if (Boolean.parseBoolean(args[2])) {
+
+			File localEiffelRepoPath = new File(System.getProperty(EiffelConstants.USER_HOME) + "\\" + EiffelConstants.EIFFEL);
+
+			// Clone Repo from GitHub
+			cloneEiffelRepo(args[0], args[1], localEiffelRepoPath);
+
+			// Read and Load JsonSchemas from Cloned Directory
+			LocalRepo LocalRepo = new LocalRepo(localEiffelRepoPath);
+			LocalRepo.readSchemas();
+			
+			ArrayList<String> jsonEventNames = LocalRepo.getJsonEventNames();
+			List<File> jsonEventSchemas = LocalRepo.getJsonEventSchemas();
```

Comment:
Wopuldn't it improve if the LocalRepo.getJsonEventSchemas would return a List<SchemaFile>?. Then you would call schemaFile.modify(jsonEventNames) instead?